xGame/client/src/Components/Match/MatchPlayerStats.js

```diff
@@ -0,0 +1,94 @@
+import React, { useState } from "react";
+import Grid from "@material-ui/core/Grid";
+import PropTypes from "prop-types";
+import "./MatchStyle.css";
+import {
+  getPieceList,
+  showTable,
+  createBoardPiece,
+} from "./HelpfulMatchTools.js";
+
+const MatchPlayerStats = (props) => {
+  const playerName = props.playerName;
+  const activeColor = props.activeColor;
+  const turnCounter = props.turnCounter;
+  const boardState = props.boardState;
+  const opponent = props.opponent;
```

Comment:
I'd suggest changing this variable name to something like "isOpponent" so that it's clear this is a boolean value

Reply:
Yeah that is much clearer thanks